cpu/samd21/periph/cpuid.c

```diff
@@ -0,0 +1,17 @@
+
+#include <string.h>
+
+#include "cpu-conf.h"
+#include "periph/cpuid.h"
+
+#define Word0 (*(volatile uint32_t *)0x0080A00C)
```

Comment:
Another remark: usually these addresses are defined in some header, e.g. `cpu/samd21/include/component/component_cpuid.h`